ganttproject/src/net/sourceforge/ganttproject/document/DocumentCreator.java

```diff
@@ -207,7 +207,9 @@ public static Runnable createAutosaveCleanup() {
       GPLogger.log("Options file:" + optionsFile.getAbsolutePath());
       BasicFileAttributes attrs = Files.readAttributes(optionsFile.toPath(), BasicFileAttributes.class);
       FileTime accessTime = attrs.lastAccessTime();
-      cutoff = Math.min(accessTime.toMillis(), now);
+      FileTime modifyTime = attrs.lastModifiedTime();
+      long lastFileTime = Math.max(accessTime.toMillis(),modifyTime.toMillis());
```

Comment:
please use spaces to separate function arguments.
```suggestion
      long lastFileTime = Math.max(accessTime.toMillis(), modifyTime.toMillis());
```